### PR TITLE
*: fix tsan in Pipeline Agg

### PR DESCRIPTION
**Pull request by ywqzzy**
### What problem does this PR solve?

Issue Number: ref #6518 

Problem Summary:
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-unit-tests/detail/tiflash-ghpr-unit-tests/8514/pipeline/ fix unstable test

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Operators/AggregateContext.cpp [hunk 1]
```diff
@@ -44,7 +44,7 @@ void AggregateContext::executeOnBlock(size_t task_index, const Block & block)
     threads_data[task_index].src_rows += block.rows();
 }
 
-void AggregateContext::writeSuffix()
+void AggregateContext::readPrefix()
```
Comment (SeaRise) [inline dbms/src/Operators/AggregateContext.cpp:47]: how about 'initConvergentPrefix'

Comment (SeaRise) [reply dbms/src/Operators/AggregateContext.cpp:47]: And how about renaming executeOnBlock to buildOnBlock

**Review by SeaRise — COMMENTED** [review]
Others LGTM

**Review by SeaRise — APPROVED** [review]

**Review by JaySon-Huang — APPROVED** [review]

**Pull request merged**